.travis.yml

```diff
@@ -1,11 +1,10 @@
+language: python
+
 sudo: false
 
 cache:
-  directories:
-    - $HOME/.ccache
-    - $HOME/.cache/pip
-    - $HOME/.cache/matplotlib
-    - $HOME/Library/Caches/pip
+  - ccache
```

Comment:
It's not clear from the travis docs that `pip` caches `$HOME/Library/Caches/pip` which is the pip cache directory on OSX.